clients/fides-js/src/components/fides.css

```diff
@@ -1113,6 +1124,11 @@ div#fides-overlay-wrapper .fides-toggle .fides-toggle-display {
 .fides-tcf-banner-container #fides-privacy-policy-link {
   margin: auto;
 }
+@media screen and (max-width: 768px) {
+  .fides-tcf-banner-container #fides-banner {
+    padding: 24px;
+  }
+}
```

Comment:
since the TCF specific styles are below the mobile media query styles, the mobile adjustment was being overridden and not being applied. This fix does the same thing, but specific to TCF padding.